cli/runtime/config/contexts.go

```diff
@@ -0,0 +1,355 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package config
+
+import (
+	"fmt"
+
+	"gopkg.in/yaml.v3"
+
+	configapi "github.com/vmware-tanzu/tanzu-framework/cli/runtime/apis/config/v1alpha1"
+	"github.com/vmware-tanzu/tanzu-framework/cli/runtime/config/nodeutils"
+)
+
+func GetContext(name string) (*configapi.Context, error) {
+	node, err := GetClientConfigNode()
+	if err != nil {
+		return nil, err
+	}
+	return getContext(node, name)
+}
+
+// Deprecated:- Use SetContext
+//nolint:all
+func AddContext(c *configapi.Context, setCurrent bool) error {
```

Comment:
Considering `Contexts` functionality is under feature-flag which is still not enabled yet, we should be able to remove this public API without Deprecating it.

Reply:
removed the deprecated tag